soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -20,6 +29,9 @@ config SYS_CLOCK_TICKS_PER_SEC
 config ARCH_HAS_CUSTOM_BUSY_WAIT
 	default y
 
+config SYS_POWER_MANAGEMENT
```

Comment:
Some of system power management policies uses timing information which is not available when there is no system timer. There could be compilation/runtime errors if you enable PM without timer.

Reply:
That already selects `TICKLESS_IDLE` which requires a timer, so this should be fixed in the system power management dependency tree if anything